platform/ui/src/components/StudyBrowser/StudyBrowser.tsx

```diff
@@ -140,7 +140,10 @@ StudyBrowser.propTypes = {
               imageSrc: PropTypes.string,
               imageAltText: PropTypes.string,
               seriesDate: PropTypes.string,
-              seriesNumber: PropTypes.string,
+              seriesNumber: PropTypes.oneOfType([
```

Comment:
This allows viewing more studies without having long stack traces.

Reply:
This is a comment in general: can't we just put these valuable comments (that are in Github pull request) as part of the code comment? If they need to be explained for a reviewer, a code reader benefits from the comment as well. You don't need to add the comment for this PR, but talking generally.